script_01.py

```diff
@@ -0,0 +1,71 @@
+import math
+import cmath
+
+def task1(text="Count stuff in this random sentence."):
+    print(text)
+    vowels=['a','e','i','o','u','y']
+    consonants=list('bcdfghjklmnpqrstvwxyz')
+    sumofv=0
+    sumofc=0
+    for i in vowels:
+        print(str(i)+': '+str(text.count(i,0,-1)))
+        sumofv=sumofv+text.count(i,0,-1)
+    print('Sum of vowels:' +str(sumofv))
+    for a in consonants:
+        sumofc=sumofc+text.count(a,0,-1)
+    print('Sum of consonants:' + str(sumofc))
+    print('Number of words: ' + str(len(text.split(' '))))
+
+def task2(minval=1,maxval=100):
```

Comment:
please add documentation to your functions and describe what are they doing. You can use this template:
```
def fun():
"""Function fun responsible for ..."""

```